vars/infra.groovy

```diff
@@ -1,5 +1,13 @@
 #!/usr/bin/env groovy
 
+Boolean isRunsOnJenkinsAzure() {
```

Comment:
I was thinking about that, but I concluded that we may change the infra layout at some point (e.g. we move to JustAnotherCloudProvider). The `buildPipeline()` logic is specific to Azure, so I concluded it's better to have "Azure" in the method name